libs/@local/hash-graph-sdk/node/src/entity.ts

```diff
@@ -0,0 +1,14 @@
+import type { Entity as GraphApiEntity } from "@local/hash-graph-client/api";
+import type { EntityRecordId } from "@local/hash-subgraph";
+
+export class Entity {
+  #entity: GraphApiEntity;
+
+  constructor(entity: GraphApiEntity) {
+    this.#entity = entity;
+  }
+
+  public get id(): EntityRecordId {
+    return this.#entity.metadata.recordId as EntityRecordId;
+  }
```

Comment:
I appreciate this is just a first function but we never use `recordId` as a whole so I'd make it `entityId` instead (and qualify `id` since it has many of them)
```suggestion
  public get entityId(): EntityRecordId {
    return this.#entity.metadata.recordId.entityId as EntityId;
  }
```